redash/handlers/data_sources.py

```diff
@@ -75,8 +75,10 @@ def post(self):
 
 class DataSourceSchemaAPI(BaseResource):
     def get(self, data_source_id):
+        refresh = request.args.get("refresh", "0", type=bool)
```

Comment:
I thought it was smarter than it was. Will hard code it to 0 and 1.

On Sun, Dec 20, 2015, 13:26 Arik Fraimovich <notifications@github.com>
wrote:

> In redash/handlers/data_sources.py
> <https://github.com/getredash/redash/pull/715#discussion_r48102448>:
>
> > @@ -75,8 +75,10 @@ def post(self):
> >
> >  class DataSourceSchemaAPI(BaseResource):
> >      def get(self, data_source_id):
> > +        refresh = request.args.get("refresh", "0", type=bool)
>
> I don't think this works the way you think it works. From what I've seen,
> if no refresh value is passed, it will return "0" (as string), which is a
> truth value... also, calling bool on any string value (including 0) will
> return True.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/getredash/redash/pull/715/files#r48102448>.
>